docs/other/dotnet.md

```diff
@@ -5,44 +5,55 @@ TOCTitle: .NET Core
 ContentId: AFFD7BDB-925E-4D02-828D-4E14360C70DA
 PageTitle: .NET Core and Visual Studio Code
 DateApproved: 2/10/2018
-MetaDescription: Visual Studio Code has great support for writing and debugging .NET Core applications.
+MetaDescription: Get started writing and debugging .NET Core apps with Visual Studio Code.
 MetaTags:
 - .NET Core
 - ASP.NET Core
 - C#
 ---
 # Using .NET Core in Visual Studio Code
 
-[.NET Core](https://docs.microsoft.com/dotnet/articles/welcome) gives you a blazing fast and modular platform for creating server applications that run on Windows, Linux and macOS. Use Visual Studio Code with the C# extension to get a powerful editing experience with full support for C# IntelliSense (smart code completion) and debugging.
+[.NET Core](https://docs.microsoft.com/dotnet/articles/welcome) provides a fast and modular platform for creating server apps that run on Windows, Linux, and macOS. Use Visual Studio Code with the C# extension to get a powerful editing experience with [C# IntelliSense](https://docs.microsoft.com/visualstudio/ide/visual-csharp-intellisense) (smart code completion) and debugging.
 
-## Getting Started
+## Get started
```

Comment:
This should be Prerequisites, and the first line can just be "Install the following:"